.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ integration, master, release-* ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ integration ]
+  schedule:
+    - cron: '16 7 * * 4'
```

Comment:
Can you add a comment indicating what interval this is? I don't think the cron syntax is intuitive to anyone

Reply:
lol beats me, this was generated, i would have to work it out myself!